### ISSUE TITLE
Feedme user import not assigning groups even when the log shows it attempting to assign the group

### ISSUE THREAD

**Issue by marcushouse** [craftcms/feed-me#598]
### Description
I'm importing users into Craft via Feedme . I've got User Groups set to "Use default value" and have selected the Default Value group.

When I run the feed, the users are created and updated, but the group is not assigned. In the log I can even see it trying to set the group with it showing ...  ,"email":"blah@nowhere.com","groups":["2"]}, but no groups are set for anyone.

What am I doing wrong? 

### Additional info
- Plugin version: 4.1.2
- Craft version: 3.3.0.1

**Comment by marcushouse** [comment]
Hmm, interestingly, after the import, I then went to users, could see no one had the correct group set. Then I went to the feed settings and hit the "Debug" option. The debug option showed very valid looking array output including the group information.

When I went back to the users list, the groups had all been applied. So somehow, the Debug option seems to have fixed the issue.

All very weird.
